IanGRap/Squish
# First Playable Final Adjustments

Contributors: Austin

Goal: Tweak the game slightly according to playtest criticism before Marcelo's crit today.

What changed:
- Increased block spawning rate
- Increase the size of lobby UI
- Increase the icon tween when a player presses ready in the lobby

How to test:
- Load up the lobby and ensure the UI is more noticible
- Load up the match scene and ensure that the blocks spawn faster
